src/code/engine_library/getDefaultVPCPopulationSettings.m

```diff
@@ -1,89 +1,183 @@
+
+% zoom int time axes
+Timeprofile.timelimit = [];
+
+% set scale of y axes
+Timeprofile.yScale = {'lin','log'};
+
+% get Types of Plot
+Timeprofile.plotTypes = {'yVsTime','resVsTime','resVsY','predVsObs','histRes','qqPlotRes'};
 
 
 return
 
-function demographic = addVPCDemographic(PopRunSet,flag)
+function PhysProperties = addVPCPhysProperties(PopRunSet,flag)
 
 switch flag
     
-    case 'default'
+    case {'parallelComparison','ratioComparison'}
         
-        demographic.name = 'demographic';
-        demographic.reportName = '';
+        PhysProperties.name = '';
+        PhysProperties.reportName = '';
         
         % list demographic properties which should be plotted
-        % pathID, reortname, display unit
-        demographic.yList = {'Organism|Age', 'Age', 'year(s)';...
-            'Organism|Weight', 'Body weight', 'kg';...
-            'Organism|Height', 'Height', 'cm';...
-            'Organism|BMI', 'BMI', 'kg/m�'};
+        % pathID, reortname, display unit,categorial text
+        PhysProperties.yList = {'Organism|Age', 'Age', 'year(s)',{};...
+            'Organism|Weight', 'Body weight', 'kg',{};...
+            'Organism|Height', 'Height', 'cm',{};...
+            'Organism|BMI', 'BMI', 'kg/m�',{};...
+            'Gender', 'Gender', '',{'Male','Female'}};
         
         % list propety which shall be displayed on the x-axes, if empty a
         % histogramm will be created
-        demographic.xList = {};
+        PhysProperties.xList = {};
         
         % list indices of popSet for that the VPC should be done
         [~,ix] = unique({PopRunSet.popcsv},'stable');
-        demographic.ixOfPopRunSets =  ix;
+        PhysProperties.ixOfPopRunSets =  ix;
         
         % how to handle PopRunSets:
         % merged populations are merged,
         % serial, for each PopRunSet an extra plot is generated
-        demographic.popRunSetMerger =  'serial';
+        PhysProperties.popRunSetMerger =  'serial';
         
         % name of Reference Population
-        demographic.ixPopRunSetRef = [];
+        PhysProperties.ixPopRunSetRef = [];
         
     case 'pediatric'
 
-        demographic.name = 'pediatricPopulation';
-        demographic.reportName = 'virtual pediatric population';
+        PhysProperties.name = 'pediatricPopulation';
+        PhysProperties.reportName = 'virtual pediatric population';
 
         
         % list demographic properties which should be plotted
         % pathID, reortname, display unit
-        demographic.yList = {'Organism|Weight', 'Body weight', 'kg';...
-            'Organism|Height', 'Height', 'cm';...
-            'Organism|BMI', 'BMI', 'kg/m�'};
+        PhysProperties.yList = {...
+            'Organism|Weight', 'Body weight', 'kg',{};...
+            'Organism|Height', 'Height', 'cm',{};...
+            'Organism|BMI', 'BMI', 'kg/m�',{};...
+            'Gender', '', '',{'Male','Female'};...
+            '<addOntogenyFactor>','','',{}};
```

Comment:
if the code finds the key word <addOntogenyFactor> in ths cell array then all ontogeny factors included in the xml file are added to this structure automatically during initialization of the workflow.